### DISCUSSION TITLE
Icon request: nas

### DISCUSSION THREAD

**Discussion by somethingSTRANGE** [FortAwesome/Font-Awesome#21118] [Icon Requests]
**What can it be used for?**

Used to represent a network attached storage (NAS) device, such as those offered by Synology, QNAP, Buffalo, and other hardware manufactures.

Used along with other icons in the "computer" category, such as "router", "server", and "hdd" shown here.

> ![fa-router-server-hdd](https://user-images.githubusercontent.com/6905832/110687171-16596680-8195-11eb-98ca-dec5b57b0ef7.png)

While "server" or "hdd" might be acceptable substitutes for a dedicated NAS icon, they're not entirely accurate. It would be nice to have an icon that explicitly represents network attached storage (file server with multiple HDDs, often in some type of RAID).

-----
**Example image (optional)**

NAS devices are often represented with icons like these:

> ![fa-nas](https://user-images.githubusercontent.com/6905832/110689815-ff684380-8197-11eb-89bf-b296f5017fef.png)

-----
**Icon request checklist**
- [x] This is a single icon or matched pair (Ex: `lock` / `unlock`) - _Single icon that complements existing icons._
- [x] The title starts with "Icon request: " and is followed by the requested icon name (Ex: `Icon request: magical-unicorn`)
- [x] I have [searched for existing issues](https://github.com/FortAwesome/Font-Awesome/issues) and to the best of my knowledge this is not a duplicate
- [x] I have included at least one example image if it could be helpful (optional)
- [x] The request is for a concrete object (it's harder to make an icon to represent happiness, it's easier to make a smiley face. ☺)

**Comment by andrewcrook** [comment]
+1  

Some sort of NAS looking icon maybe based on a Synology box 
There is also opertunity to add muliple devices with different number of bays

**Comment by sensibleworld** [comment]
This is a quick update to let folks know that we’re refining our icon request workflow behind the scenes, and as part of that we’re going through all open requests and filing them into several new buckets to be addressed.

In this case we’ve added this issue to our new list to be created! We can’t promise any firm timelines right now, but this will help us keep track of it.

We’ll update this issue when the icon is released into the wild 🙂

**Comment by sensibleworld** [comment]
Added to 7.2: https://fontawesome.com/icons/nas?f=classic&s=solid
